### DISCUSSION TITLE
Launch Program?

### DISCUSSION THREAD

**Discussion by jnew1213** [Lissy93/dashy#935] [Help and Support] [answered]
New to Dashy. Loving it so far (two days now). Running the latest version in a Docker container. Something I can't find after hours of searching...

Can I launch a program from Dashy?

I would like to launch a number of programs, including mstsc.exe, the Microsoft Remote Desktop client, and vmware-view.exe, the Horizon Client. Both take command line arguments to seamlessly open the app and connect to a destination host. I use both of these, and several more, many times per day.

Is it possible to launch a program from Dashy? If so, how?

**Comment by lissy93** [comment, marked as answer]
The answer is yes and no.

Natively it's not possible for a website (like Dashy)  to launch an app on the host system (without disabling some browser security, I think `--allow-file-access-from-files` <sup><a href="http://www.chrome-allow-file-access-from-file.com/">Source</a></sup> - not recommend!)

However there's no reason you couldn't write a script that's hosted somewhere, and then trigger that from a GET request / HTTP hook within Dashy. That should be possible once Action Buttons are implemented, but that's not finished yet.

**Reply by jnew1213** [reply]
Thank you for the quick response. I'll leave Dashy to handle the bulk of my access needs and curate a few on-screen icons to handle a couple of program launches.

Warm regards.
